src/schemas/json/project.json

```diff
@@ -241,6 +244,7 @@
 		},
 
 		"commands": {
+            "description": "Commands that are available for this application.",
```

Comment:
Commands are not going to be present in RC2 due to the move to dotnet-cli. There will be concepts to help replace them (project-specific tools, etc.) but it's going to be different enough to warrant it's own docs.